Ska/engarchive/time_adjust.py

```diff
@@ -0,0 +1,117 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Describe what this module does.
+"""
+
+import copy
```

Comment:
Imports ordered in sections: standard library imports then other installed packages.